src/bloxone_client/api_client.py

```diff
@@ -215,6 +231,12 @@ def param_serialize(self,
 
         # body
         if body:
+            if len(self.configuration.default_tags) > 0 and hasattr(body, 'tags'):
+                if body.tags is None:
+                    body.tags = {}
+                for k, v in self.configuration.default_tags.items():
+                    if k not in body.tags:
+                        body.tags[k] = v
```

Comment:
I would it to be a class method

Reply:
Added a class method called `add_default_tags`